src/group_theory/coset.lean

```diff
@@ -295,6 +295,14 @@ instance (s : subgroup α) : inhabited (quotient s) :=
 protected lemma eq {a b : α} : (a : quotient s) = b ↔ a⁻¹ * b ∈ s :=
 quotient.eq'
 
+@[to_additive quotient_add_group.eq']
+lemma eq' {a b : α} : (mk a : quotient s) = mk b ↔ a⁻¹ * b ∈ s :=
+quotient_group.eq
```

Comment:
It looks to me like this file can't decide whether `(mk a : quotient s)` or `(↑a : quotient s)` is the sump normal form. Do you have a feeling for which would be best? I think if you had a simp lemma rewriting one to the other then we wouldn't need both forms.

Reply:
I prefer `mk a`, because sometimes you don't need to write `: quotient s`. Whereas with `↑a`, you always have to write `: quotient s`.

Reply:
Maybe we should just remove the coercion altogether. Perhaps worth asking on Zulip?

At any rate, we may as well put this in to unblock you, but it would be nice to follow-up with removing all the duplicate lemmas using the alternate spelling.

bors merge